README.md

```diff
@@ -0,0 +1,43 @@
+# OpenTelemetry Auto Instrumentation
+
+This project provides [OpenTelemetry] instrumentation for [Go] libraries using [eBPF].
+
+## Project Status
+
+:construction: This project currently a work in progress.
+
+### Compatability
+
+This project ensures compatibility with the current supported versions of the [Go language](https://golang.org/doc/devel/release#policy):
+
+> Each major Go release is supported until there are two newer major releases.
+> For example, Go 1.5 was supported until the Go 1.7 release, and Go 1.6 was supported until the Go 1.8 release.
+
+For versions of Go that are no longer supported upstream, this project will stop ensuring compatibility with these versions in the following manner:
+
+- A minor release will be made to add support for the new supported release of Go.
+- The following minor release will remove compatibility testing for the oldest (now archived upstream) version of Go.
+   This, and future, releases may include features only supported by the currently supported versions of Go.
+
+Currently, this project is tested for the following environments.
+
+| OS      | Go Version | Architecture |
+| ------- | ---------- | ------------ |
+| Ubuntu  | 1.19       | amd64        |
+| Ubuntu  | 1.18       | amd64        |
+
+This project is known to not work in Windows environments.
```

Comment:
Regarding Windows, I think it can, however I'm not sure what's current status of the project. In this blog post it's mentioned that it's in early stage of development https://cloudblogs.microsoft.com/opensource/2021/05/10/making-ebpf-work-on-windows/, but that was more than year ago, so situation might be a little bit different now. 
There were some attempts to implement/port ebpf to FreeBSD for instance, but I'm not sure if there exists any production ready (or beta at least) implementation.

Reply:
How about the following?

```suggestion
OpenTelemetry Go Automatic Instrumentation requires Linux kernel ≥ 4.4.
```